components/net/image_cache_thread.rs

```diff
@@ -551,18 +544,13 @@ impl ImageCache {
                                 FetchResponseMsg::ProcessRequestBody |
                                 FetchResponseMsg::ProcessRequestEOF => return,
                                 FetchResponseMsg::ProcessResponse(meta_result) => {
-                                    ResponseAction::HeadersAvailable(meta_result.map(|m| {
-                                        match m {
-                                            FetchMetadata::Unfiltered(m) => m,
-                                            FetchMetadata::Filtered { unsafe_, .. } => unsafe_
-                                        }
-                                    }))
+                                    FetchResponseMsg::ProcessResponse(meta_result)
                                 }
                                 FetchResponseMsg::ProcessResponseChunk(new_bytes) => {
-                                    ResponseAction::DataAvailable(new_bytes)
+                                    FetchResponseMsg::ProcessResponseChunk(new_bytes)
                                 }
                                 FetchResponseMsg::ProcessResponseEOF(response) => {
-                                    ResponseAction::ResponseComplete(response)
+                                    FetchResponseMsg::ProcessResponseEOF(response)
```

Comment:
I have already fixed the redundant code.
Do I have to pull-request again for fixing the redundant code?